php/class-wp-cli.php

```diff
@@ -21,11 +48,77 @@ public function create_guest_authors( $args, $assoc_args ) {
 		global $coauthors_plus;
 
 		$defaults = array(
-				// There are no arguments at this time
-			);
+			'roles' => array( 'administrator', 'editor', 'author', 'contributor' ),
+			'batch-size' => 1000,
```

Comment:
Why would folks need to specify this? What would be the difference if we were to limit it to batches of 100 at a time?

Reply:
It's to stop the script from fetching **ALL** of the users at once and crashing servers with low memory limits, while also allowing servers with greater resources to open up the throttle.

Reply:
If we were to limit it to batches of 100 at a time, it would take longer but use less memory (a tradeoff I've had to make many times in the past running on low-end equipment, haha)